### DISCUSSION TITLE
Node.js heap out of memory

### DISCUSSION THREAD

**Discussion by mrt-xci** [verdaccio/verdaccio#3765] [Q&A] [answered]
Hi,
Im running Verdaccio 5.15 docker version. and getting the error "[Node.js heap out of memory]"
is there anyway from docker-compose to increase heap memory? any env for this?

/Morten

**Comment by juanpicado** [comment, marked as answer]
Probably yes but maybe you find the answer in a Docker forum faster. It should be the same for any other sort of container.
